src/Microsoft.DotNet.Wpf/tests/UnitTests/WindowsBase.Tests/System/Windows/SplashScreenTests.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Windows;
+
+public class SplashScreenTests
+{
+    [Fact]
+    public void Constructor_NullImageSource_ThrowsArgumentNullException()
+    {
+        Assert.Throws<ArgumentNullException>(() => new SplashScreen(null));
+    }
+
+    [WpfFact]
```

Comment:
Here is the sample `[WpfFact]` test. This will pop and close the splash screen. With a normal `[Fact]`  it would fail when attempting to dispatch.